src/app/pages/courses/course-list/course-list.component.html

```diff
@@ -2,8 +2,8 @@
   <li *ngFor="let course of courses">
     <app-course-item
       [course]="course"
-      (courseEdit)="onEditCourse(course)"
-      (courseDelete)="onDeleteCourse(course)">
+      (courseEdit)="onEditCourse(course.id)"
```

Comment:
Here should be `onEditCourse($event)`

Or you can leave `onEditCourse(course.id)` and make `this.courseEdit.emit()` without value